core/src/main/java/nl/nn/adapterframework/management/bus/BusMessageUtils.java

```diff
@@ -79,4 +97,30 @@ public static Response convertToJaxRsResponse(Message<?> response) {
 
 		return builder.build();
 	}
+
+	public static UserDetails getUserDetails() {
+		Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+		if(authentication != null && authentication.getPrincipal() instanceof UserDetails) {
+			return (UserDetails) authentication.getPrincipal();
+		}
+		return null;
+	}
+
+	/**
+	 * See AuthorityAuthorizationManager#ROLE_PREFIX
+	 */
+	public static boolean hasAuthority(String role) {
```

Comment:
Dit moet zijn `hasAuthority(String authority)` of `hasRole(String role)`, maar niet door elkaar.